app/precompiled.py

```diff
@@ -512,9 +512,14 @@ def _get_out_of_bounds_pages(src_pdf_bytes):
 
 
 def escape_special_characters_for_regex(string):
-    special_characters = ["[", "^", "$", ".", "|", "?", "*", "+", "(", ")"]
+    # those characters perform functions in regex expressions and have to be escaped:
+    special_characters = ["[", "{", "^", "$", ".", "|", "?", "*", "+", "(", ")"]
     for character in special_characters:
         string = string.replace(character, r"\{}".format(character))
+
+    bad_slashes = re.findall(r"\\[a-mo-rt-zA-Z]", string)
+    for s in list(set(bad_slashes)):
```

Comment:
Tests seem to pass without the `set`, I think you just need `list(bad_slashes)` here

Reply:
Ahh, that's the case of missing test case :D. It is a set, because in a set values are unique, so we don't end up with too many backslashes :)

Reply:
I added the missing test case now.